.github/CONTRIBUTING.md

```diff
@@ -333,50 +374,45 @@ Here are some contribution quick tips:
+assets:
 
-Great question! Contribution does not require creating or maintaining our
-assets. Here are some other ways you can contribute, which don't require much
-work:
+<!--
+TODO: Are these the internal Slack channels? If so, why are they listed here?
+-->
 
-- **Join our slack community and interact with other users.** We have hundreds
-  of users world wide, and quite a small team in comparison. While we try our
-  best to answer questions on slack, it's not always possible to answer
-  everything. One of the easiest ways to help us it to jump in on slack
-  conversations if there's something you know the answer to! Some of our most
-  popular slack channels include `#carbon-react`, `#carbon-web-components`,
-  `#carbon-design-system`, `#carbon-ng`, `#carbon-vue`, `#carbon-announcements`.
-- **Report bugs.** Even if you don't have the time to contribute a bug fix,
-  opening an issue alone makes a big difference! Be sure to completely fill out
-  the issue template to best help us understand what is going wrong.
+- **Join our Slack community:** Engage with hundreds of users in channels like
+  `#carbon-react`, `#carbon-web-components`, `#carbon-design-system`,
+  `#carbon-ng`, `#carbon-vue`, and `#carbon-announcements`.
+- **Report bugs:** Even if you can’t submit a fix, opening a well-documented
+  issue makes a big difference.
 
-### If I'm contributing code, am I required to contribute it for all frameworks?
+### Do I need to submit code for every framework?
 
-If you've been working with Carbon for a while, you know that we have code
-assets in multiple frameworks. While the core team only maintains the React and
-Web Component implementations, we work closely with the other framework teams to
-maintain parity. If you're contributing a bug fix in a framework other than
-React for a component which also exists in the React and Web Components version,
-you can either contribute a fix for the that version as well or open an issue so
-that we can update the component accordingly.
+While the core team maintains the React and Web Component implementations, other
+framework teams work to maintain parity. If you fix a bug in one framework for a
+component that exists in multiple frameworks, either submit a fix for the other
+versions or open an issue so we can update them.
 
-### How can I test-drive components or provide a bug reproduction?
+### How can I test components or share a bug reproduction?
 
-There are some fantastic tools out there today that provide a near fully-fledged
-development environment in the browser. These are incredibly helpful for trying
-out simple configurations or providing working code demonstrating bugs or
-problems with the libraries.
+<!--
+TODO: Should
+`[Carbon Design System Sandboxes](https://github.com/carbon-design-system/sandboxes)`
+be somewhere in here? If so, should it be its own thing or should it replace one
+or both of the `Stackblitz` and `Codesandbox` sections below?
+-->
```

Comment:
Codesandbox should be removed. Adding sandboxes is a good idea. We keep the `examples/vite` one around for easy local testing/exploring and it's been around for a long time so it's bookmarked in lots of places that I don't want to have to update 😅